.travis.yml

```diff
@@ -20,14 +20,14 @@ go:
 env:
 - GORACE="halt_on_error=1"
   BUILD_VERSION=$(echo ${TRAVIS_COMMIT} | cut -c 1-10)
-  TM_VERSION=v0.21.0
+  TM_VERSION=v0.25.0
```

Comment:
There is now an iov-one/tendermint build on dockerhub? Or using the official tendermint/tendermint images?

Reply:
This is used to built the tendermint binary in L30 for integration tests in `start_test.go`. Not docker, yet